tests/testthat/test-generate_qc_report.R

```diff
@@ -5,8 +5,31 @@ filt_sce <- filter_counts(sce)
 test_that("generating a qc report works", {
   qc_file <- generate_qc_report(library_id = "TEST",
                                 unfiltered_sce = sce,
-                                filtered_sce = filt_sce, )
+                                filtered_sce = filt_sce)
   expect_true(file.exists(qc_file))
   # clean up
   file.remove(qc_file)
 })
+
+test_that("generating a qc report works with a different template", {
+
+  test_template <- system.file(file.path("inst", "rmd", "test_report.rmd"), package = "scpcaTools")
```

Comment:
I think you actually want to put this file (since it is only for testing) in _this_ directory (`tests/testthat`) and then get it with just:
```suggestion
  test_template <-  "test_report.rmd"
```